### PR TITLE
Add a script to generate coverage results.

### PR DESCRIPTION
**Pull request by dlorenc**

### CODE REVIEW

**Comment by jimmidyson** [comment]
How about integration with coveralls.io?

**Comment by dlorenc** [comment]
@minikube-bot retest this please

File: hack/cover.sh [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -e
+
+for pkg in $(go list ./pkg/minikube/...); do
+    go test -v $pkg --coverprofile=out/coverage.out
+    go tool cover -html out/coverage.out
+done
```
Comment (luxas) [inline hack/cover.sh]: nit: newline

**Comment by luxas** [comment]
Maybe we want to add this to `make test`

**Comment by aaron-prindle** [comment]
LGTM

**Pull request merged**